lib/dynamo-backup.js

```diff
@@ -55,8 +55,17 @@ DynamoBackup.prototype.backupTable = function (tableName, backupPath, callback)
         stream:     stream
     });
 
+    uploader.on('failed', function(err) {
+        console.log("Upload of table "+tableName+" failed with error: "+err);
```

Comment:
@rambler - the `console.log` doesn't belong here.  An `error` event could be emitted from here, but I've tried to keep console logs out so people can choose how errors are displayed.